Ports/flare-engine/package.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='flare-engine'
+useconfigure='true'
+version='1.14'
+depends=(
+    'SDL2'
+    'SDL2_image'
+    'SDL2_mixer'
+    'SDL2_ttf'
+)
+files=(
+    "https://github.com/flareteam/flare-engine/archive/refs/tags/v${version}.tar.gz 2c1bafeaa5ac26c10449bfcb7151b06e8a22547aa7364d2a39bbcb135aa50a09"
```

Comment:
Btw, is there a rule for when to use single quotes or double quotes? I fixed up the simple values to use single quotes, but I see that this and a couple of other parts use double quotes (even in the documentation example and other ports)

Reply:
Single quotes keep variable expansion from happening, while double quotes (or no quotes at all) don't. To make it clear that something is a string, we have started to quote everything. Whether a string gets single quotes or double quotes depends on whether it has a variable in it.